### PR TITLE
Feat: added UI circles

### PR DESCRIPTION
**Pull request by Pr0bka**

### CODE REVIEW

File: src/components/UserSquare.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import classes from './UserSquare.module.css';
+
+const UserSquare = (props) => {
+  const [isClicked, setIsClicked] = useState(false);
```
Comment (AntonOvc) [inline src/components/UserSquare.js:5]: probably `isActive`?

File: src/components/UserSquare.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
```
Comment (AntonOvc) [inline src/components/UserSquare.js:1]: please move this file and css module to components/${componentName} folder

**Review by AntonOvc — CHANGES_REQUESTED** [review]
Looks awesome! but please pay more attention to naming

File: src/components/Header.module.css [hunk 3]
```diff
@@ -52,6 +48,29 @@ button {
...
+    font-size: xxx-large;
+    color: white;
+    display: flex;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    justify-content: center;
+   
+  }
+
+  .firsDiv {
```
[the remark this answers is not in the document]
Comment (Pr0bka) [reply src/components/Header.module.css]: Fixed

File: src/components/Header.module.css [hunk 4]
```diff
@@ -52,6 +48,29 @@ button {
...
+    display: flex;
+    justify-content: center;
+    margin-bottom: 20px;
+  }
+
+  .secondDiv {
```
[the remark this answers is not in the document]
Comment (Pr0bka) [reply src/components/Header.module.css]: Fixed

File: src/components/UserSquare.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+import classes from './UserSquare.module.css';
+
+const UserSquare = (props) => {
```
[the remark this answers is not in the document]
Comment (Pr0bka) [reply src/components/UserSquare.js]: Fixed

File: src/components/UserSquare.js (hunk 1 above)
Comment (Pr0bka) [reply src/components/UserSquare.js]: Fixed

File: src/components/UserSquare.js (hunk 2 above)
Comment (Pr0bka) [reply src/components/UserSquare.js]: Fixed

**Review by AntonOvc — APPROVED** [review]
lgtm 👍

**Pull request merged**